.github/workflows/build.yml

```diff
@@ -0,0 +1,325 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: "0 7 * * *"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  azCliVersion: 2.6.0
+  ref_javaee: 6addd99d8bc3f472e040f11c053a37e1ac370229
```

Comment:
Where do we get this id from and when this needs to be updated?

Reply:
@git4rk  the `ref_javaee` is the git commit hash of the commit from [Azure/azure-javaee-iaas](https://github.com/Azure/azure-javaee-iaas) repo to use.  In this case, [6addd99](https://github.com/Azure/azure-javaee-iaas/commit/6addd99d8bc3f472e040f11c053a37e1ac370229).  We are essentially using the commit hash as a tag, because, in fact, that is what a lightweight tag is.